include/controllers/MessagesController.h

```diff
@@ -52,7 +52,7 @@ class MessagesController : public QObject
     MessagesController(Messenger *messenger, const Settings *settings, Models *models, UserDatabase *userDatabase,
                        QObject *parent);
 
-    void loadMessages(const ChatHandler &chat);
+    void loadMessages(const ChatHandler &chat, bool isNewChat);
```

Comment:
Replace boolean flag with 2 methods.
Is this method still in use?

Reply:
Replaced and renamed.
All methods for loading are used